_people/christey_nethercott.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Christey Nethercott
+headshot: dgWk7Dw
+gender: female
+submitted: false
+
+links:
+  - type: starnow
```

Comment:
Do we support this yet? [I can't see it](https://github.com/newtheatre/history-project/blob/master/_data/link-types.yaml). Do all different types need to be defined in that yaml doc?